### DISCUSSION TITLE
Galaxy Brain

### DISCUSSION THREAD

**Discussion by onwp** [Schweinepriester/github-profile-achievements#646] [Q&A] [answered]
Do you know if terms updated for Galaxy Brain badge? 

Or is there an official page where we can see the terms for badges?

**Comment by Schweinepriester** [comment]
> Or is there an official page where we can see the terms for badges?

Afaik still no official page…

**Reply by ParanoidUser** [reply]
There are a few people (like [PopiChan](https://github.com/PopiChan?tab=achievements&achievement=galaxy-brain)) who received Galaxy Brain today by answering trivia questions. So the terms haven't changed.  

However, many cases have been reported where people did not get the badge because they marked their own answers somewhere. Possible solution might be to find & unmark those answers as described here - https://github.com/orgs/community/discussions/18454

**Comment by LauraBigoni** [comment, marked as answer]
For galaxy brain you need to suggest answers to questions posted in discussion. If your answer is helpful then the person who asked the question can mark you answer as the solution. Once you've had 2 questions marked as solutions then you'll get galaxy brain.

Here's the link to the general GitHub community discussions. It might be a good place to start. https://github.com/community

If you found this useful then please mark my answer as a solution so I can get the badge.
